addons/pos_restaurant_self_order/controllers/entry.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+
+from odoo.addons.pos_self_order.controllers.entry import PosQRMenuController
+from odoo.addons.pos_restaurant_self_order.controllers.utils import get_table_sudo
+
+from odoo.http import route
+from odoo.http import request
+
+
+class PosRestQRMenuController(PosQRMenuController):
```

Comment:
maybe there's a better way of doing this (also for the other controllers). We think while DAMA is testing.